### PR TITLE
feat(blockscout): add blockscout resource creation

### PR DESCRIPTION
**Pull request by servalD**
### Linked issues

- https://github.com/AshAvalanche/ash-rs/issues/95

### Changes

- Add helper for blockscout
- Add template for blockscout

### CODE REVIEW

File: crates/ash_sdk/Cargo.toml [hunk 1]
```diff
@@ -42,7 +42,7 @@ rustls-pemfile = "1.0.3"
 sha2 = "0.10.7"
 oauth2 = "4.4.2"
 url = "2.4.1"
-ash_api = { version = "0.1.5" }
+ash_api = { version = "0.1.6" , path = "/home/gaetan/Documents/hai/ash-api-rs"}
```
Comment (leopaul36) [inline crates/ash_sdk/Cargo.toml:45]: Please change this

File: crates/ash_cli/src/utils/templating.rs [hunk 2]
```diff
@@ -1156,6 +1156,33 @@ pub(crate) fn template_avalanche_subnet_props_table(
...
+            &blockscout
+                .blockscout_status
+                .clone()
+                .unwrap()
+                .running
+                .unwrap_or_default()
+        ),
+    ]);
+
+    // TODO: Add the rest of the Subnet properties
```
Comment (leopaul36) [inline crates/ash_cli/src/utils/templating.rs:1181]: ?

**Review by leopaul36 — CHANGES_REQUESTED** [review]

File: crates/ash_cli/src/console/helper.rs [hunk 3]
```diff
@@ -206,6 +212,47 @@ fn rpc_helper(
...
+            "Resource is not a `blockscout`!".to_string(),
+        ));
+    };
+
+    spinner.finish_and_clear();
+
+    println!(
+        "Blockscout URL:\n  {}",
+        type_colorize(&format!(
+            "http://{}:3000/",
```
Comment (leopaul36) [inline crates/ash_cli/src/console/helper.rs:248]:
```suggestion
            "http://{}:80",
```

**Review by leopaul36 — CHANGES_REQUESTED** [review]

File: crates/ash_cli/src/utils/templating.rs [hunk 4]
```diff
@@ -1156,6 +1156,30 @@ pub(crate) fn template_avalanche_subnet_props_table(
...
+pub(crate) fn template_blockscout_props_table(
+    blockscout: &console::api_models::GetAllProjectResources200ResponseInner,
+) -> Table {
+    let mut props_table = Table::new();
+    props_table.set_format(*format::consts::FORMAT_NO_BORDER_LINE_SEPARATOR);
+
+    props_table.add_row(row![
+        "IP address".bold(),
+        type_colorize(&blockscout.blockscout_ip.clone().unwrap_or_default()),
+    ]);
```
Comment (leopaul36) [inline crates/ash_cli/src/utils/templating.rs:1168]: This does not seem to work.

I'm getting:

```
+---------------+-------------+-----------------+---------------------+--------+------------------+---------+-------------------------------------------------------------------+
| blockscout    | 7053...982a | Blockscout      | azure/francecentral | Medium | 2024-06-06T15:07 | Running |  IP address |                                                     |
|               |             |                 |                     |        |                  |         |  Running    | true                                                |
+---------------+-------------+-----------------+---------------------+--------+------------------+---------+-------------------------------------------------------------------+
```

Whereas the `blockscoutIp` can be fetched successfully in --json mode:

```
cargo run -- console resource info blockscout --json | jq
   Compiling ash_sdk v0.4.3 (/home/leo/Ash/ash-rs/crates/ash_sdk)
   Compiling ash_cli v0.4.3 (/home/leo/Ash/ash-rs/crates/ash_cli)
    Finished dev [unoptimized + debuginfo] target(s) in 7.29s
     Running `target/debug/ash console resource info blockscout --json`
{
  "blockscoutConfig": {
    "blockscoutConfig": {}
  },
  "blockscoutIp": "20.19.244.45",
  "blockscoutStatus": {
    "restartRequired": false,
    "running": true
  },
  "cloudRegionId": "094fcf66-2cb7-416c-8e4e-5069eba0b8cd",
  "created": "2024-06-06T15:07:43.573921",
  "id": "70531311-3cb7-46ff-830a-d74824e3982a",
  "name": "blockscout",
  "ownerId": "9b80e516-5f06-4983-b955-0a5a146afa60",
  "projectId": "9af0e486-4571-4f45-816e-90140b9c3d43",
  "resourceType": "blockscout",
  "size": "medium",
  "status": "running"
}
```

**Review by leopaul36 — CHANGES_REQUESTED** [review]

File: crates/ash_cli/src/utils/templating.rs (hunk 4 above)
Comment (servalD) [reply crates/ash_cli/src/utils/templating.rs:1168]: fixed in https://github.com/AshAvalanche/jeeo/pull/155/commits/e031e60af9829b82978075f1fe13e2fedbcf9efc

**Review by leopaul36 — APPROVED** [review]

**Pull request merged**